inc/saiport.h

```diff
@@ -1032,6 +1034,15 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_GLOBAL_FLOW_CONTROL_MODE,
 
+    /**
+     * @brief Forward or not forward the global flow control frame
```

Comment:
When not forwarded, I suppose the flow control frame is terminated in the MAC.  Perhaps better to make that clear.

Reply:
yes, @Pterosaur , can you add comments to make this clear.

Reply:
the forward means, when it is gearbox, it is forward the flow control frame from one side of the port to the other side of the port, it could be lineside to the systemside or vice versa.

when it is asic, it means the flow control frame will be asic to the packet core from the serdes block and asic can process the flow control frame.

Reply:
Agree - it would be less confusing to simply say the ASIC or the port terminates and honors the flow control frame.

Reply:
@dipankar-ba - do you approve this pr or anything to be revisited?